lib/bugsnag/integrations/railtie.rb

```diff
@@ -22,7 +22,6 @@ class Railtie < Rails::Railtie
       # Skipping API key validation as the key may be set later in an
       # initializer. If not, the key will be validated in after_initialize.
       Bugsnag.configure(false) do |config|
-        config.logger = ::Rails.logger
```

Comment:
I believe the logger was set here so that any log output from Bugsnag that occurred in a config/initializers/ file would end up in the Rails log.

When trying the example at https://github.com/bugsnag/bugsnag-ruby/tree/800ea83ee4885df2e2f2c48570e13ffa7e75b9b8/example/rails-51 with your change, Bugsnag log output resulting from a file in config/initializers/ doesn't end up in log/development.log.